assets/global.js

```diff
@@ -378,12 +378,18 @@ class ModalDialog extends HTMLElement {
       'click',
       this.hide.bind(this)
     );
-    this.addEventListener('click', (event) => {
-      if (event.target.nodeName === 'MODAL-DIALOG') this.hide();
-    });
-    this.addEventListener('keyup', () => {
+    this.addEventListener('keyup', (event) => {
       if (event.code.toUpperCase() === 'ESCAPE') this.hide();
     });
+    if (this.classList.contains('media-modal')) {
```

Comment:
Not sure how I feel about using class conditions for logic. Is there any other use of this on the theme that you can remember?
 
I'm wondering if that wouldn't be a good use for `data-` attributes instead.